spdk-sys/build.sh

```diff
@@ -12,24 +12,30 @@ rm libspdk.so
 ./configure --enable-debug \
 	--target-arch=nehalem \
 	--without-isal \
-	--with-iscsi-initiator \
```

Comment:
The nixpkgs libspdk still builds `--with-iscsi-initiator`.

Reply:
Yeah, we should remove it there as well.